hack/manifest-gen/manifest-gen.sh

```diff
@@ -9,7 +9,7 @@
 set -euo pipefail
```

Comment:
I feel like we could move `manifest-gen` to the `deploy` as well, so we end up with something like:

```
deploy/helm
deploy/manifest-gen
```

What do you think?

Reply:
I considered doing that myself. However, since we don't have plans to make manifest-gen public, I think it should stay in its current location so that we don't get questions or support requests from users who mistakenly believe that it is a tool intended for general use.